test/test25.md

```diff
@@ -0,0 +1,119 @@
+# Latinisms test
+
+This test should produce 44 suggestions and seven warnings for rule #21.
+
+a priori
+
+ad hoc
+
+ad infinitum
+
+cf
+
+cf.
+
+caveat
+
+circa
+
+de facto
+
+ergo
+
+et cetera
+
+etcetera
+
+etc.
+
+etc
+
+&c.
+
+&c
+
+exempli gratia
+
+e.g.
+
+eg
+
+gratis
+
+id est
+
+i.e.
+
+ie
+
+in situ
+
+nota bene
+
+n.b.
+
+nb
+
+per
+
+per diem
+
+per capita
+
+per se
+
+pro bono
+
+proviso
+
+stanza
+
+status quo
+
+verbatim
+
+versus
+
+vs.
+
+via
+
+vice versa
+
+viz.
+
+AM
+
+PM
+
+a.m.
+
+p.m.
```

Comment:
```suggestion
5 AM

5PM

12 a.m.

12p.m.
```

I think these should be practical examples rather than just the term.

Reply:
The other common usage of AM/PM in sentences is "in the AM / PM" but this case would prevent AM and PM from being used as acronyms, which I think might be a bad call.